airflow/www/static/js/dag/details/RenderedJsonField.tsx

```diff
@@ -0,0 +1,70 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from "react";
+
+import ReactJson from "react-json-view";
+
+import { Flex, Button, Code, Spacer, useClipboard } from "@chakra-ui/react";
+
+interface Props {
+  content: string;
+}
+
+const RenderedJsonField = ({ content }: Props) => {
+  let contentJson = null;
+  let contentFormatted = "";
+  let isJson = false;
+  try {
```

Comment:
We can make this a function outside of the component and then call it inside the component.

Reply:
Thanks for the hint, adjusted in e5f6e8f6d0b501e315e393de4f3aecbc95c8bf35